server/controllers/gdpr.js

```diff
@@ -11,10 +11,11 @@ const customerDataRequest = async (topic, shop, webhookRequestBody) => {
       customer: { id, email },
       orders_requested,
     } = JSON.parse(webhookRequestBody);
```

Comment:
I think that there is not need to use `JSON.parse(webhookRequestBody)`, since in the server we are setting this ` app.use(express.json());` before GDPR routes, when the body request reach this function it is already parsed

Reply:
Oh ye! I know of other issues with this PR too, it's just a provisional one to ensure there's transparency that I'm already on it and it's actively being worked on instead of just saying "The team is aware and actively working on it" or "A fix is coming soon™" (hate that stuff)

But hey thank you so much for pointing it out!